### ISSUE TITLE
Not working on FreeBSD?

### ISSUE THREAD

**Issue by stocklassa** [mailjet/mailjet-gem#81]
My Mailjet calls are crashing on FreeBSD. The exact same script works fine on my Ubuntu desktop, but crashes as soon as I execute it on the server. 

``` ruby
require "mailjet"

Mailjet.configure do |config|
  config.api_key = "<correct key>"
  config.secret_key = "<correct private key>"
end

campaign = Mailjet::Campaign.find(<valid id>)
```

> NoMethodError: undefined method `[]' for nil:NilClass
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/api_error.rb:13:in`initialize'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/connection.rb:83:in `new'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/connection.rb:83:in`handle_exception'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/connection.rb:70:in `rescue in handle_api_call'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/connection.rb:61:in`handle_api_call'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/connection.rb:43:in `get'
>         from /home/admin/.rvm/gems/ruby-2.3.0/gems/mailjet-1.4.10/lib/mailjet/resource.rb:74:in`find'
>         from (irb):8
>         from /home/admin/.rvm/rubies/ruby-2.3.0/bin/irb:11:in `<main>'

I have the same configuration of gems on the target platform:

> activesupport (5.0.0.1)
> bigdecimal (1.2.8)
> concurrent-ruby (1.0.2)
> domain_name (0.5.20160826)
> http-cookie (1.0.3)
> i18n (0.7.0)
> io-console (0.4.5)
> json (1.8.3)
> mail (2.6.4)
> mailjet (1.4.10)
> mime-types (3.1)
> mime-types-data (3.2016.0521)
> mysql (2.9.1)
> netrc (0.11.0)
> psych (2.0.17)
> rack (2.0.1)
> rdoc (4.2.1)
> rest-client (2.0.0)
> thread_safe (0.3.5)
> tzinfo (1.2.2)
> unf (0.1.4)
> unf_ext (0.0.7.2)

And the same version of ruby

> ruby -v
> ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-freebsd10.1]

**Comment by Zhivko-Mailjet** [comment]
Hi @Feuerschwinge 

Thanks for reporting the issue. We will try to reproduce it but since we have no experience with FreeBSD, we can't guarantee that we will manage to solve it. I will let you know once we have any information.

**Comment by Feuerschwinge** [comment]
Thanks! :)

**Comment by xab3r** [comment]
Fixed in v1.6.0
